packages/database/src/core/SyncTree.ts

```diff
@@ -482,16 +501,10 @@ export function syncTreeApplyTaggedQueryMerge(
   }
 }
 
-/**
- * Add an event callback for the specified query.
- *
- * @returns Events to raise.
- */
-export function syncTreeAddEventRegistration(
-  syncTree: SyncTree,
+export function syncTreeRegisterSyncPoint(
```

Comment:
Another comment here to explain the rational for splitting this out from syncTreeAddEventRegistration. Can be short.